### PR TITLE
Discourage people from instantiating the library with a literal token

### PR DESCRIPTION
**Pull request by SimonWoolf**
another customer today instantiating the library with a token and being confused when it broke after an hour, and to be fair, the previous wording ("Use this option if you wish to use Token authentication") was kinda misleading for an option we don't want anyone to use in production

### CODE REVIEW

File: content/core-features/authentication.textile [hunk 1]
```diff
@@ -52,12 +52,14 @@ Client-side devices should generally be considered untrusted, and as such, it is
...
-* "<span lang="default">@useTokenAuth@</span><span lang="csharp">@UseTokenAuth@</span>":/realtime/types#client-options is true
+* "<span lang="default">@useTokenAuth@</span><span lang="csharp">@UseTokenAuth@</span>":/realtime/types#client-options is true;
+* a "<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span>":/realtime/types#client-options is provided (only for pre-1.1 client libraries);
+* a "<span lang="default">@token@</span><span lang="csharp">@Token@</span>":/realtime/types#client-options or "<span lang="default">@tokenDetails@</span><span lang="csharp">@TokenDetails@</span>":/realtime/types#client-options property is provided
+
+The last of those (providing a literal <span lang="default">@token@</span><span lang="csharp">@Token@</span> or <span lang="default">@tokenDetails@</span><span lang="csharp">@TokenDetails@</span>) is mostly only used for testing: since tokens are short-lived, in production you almost always want to use an authentication method that allows the client library to automatically renew the token when the previous one expires.
```
Comment (paddybyers) [inline content/core-features/authentication.textile]: To unnecessarily include a split infinitive ....

Comment (SimonWoolf) [reply content/core-features/authentication.textile]: 236a6eb :slightly_smiling_face:

Comment (paddybyers) [reply content/core-features/authentication.textile]: I hate to say this, but I think what you have now really is worse. I was thinking that " .. to renew the token automatically ..." was the obvious resolution. Now I do regret raising it. :)

**Comment by SimonWoolf** [comment]
> Although should we not just address #300 at the same time?

already addressed by https://github.com/ably/docs/pull/412

**Comment by mattheworiordan** [comment]
Thanks

**Pull request merged**